src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -545,7 +545,9 @@ public async Task Initialize(HostWaitLoop waitLoop, CancellationToken token)
 
             try
             {
-                await this.MICommandFactory.EnableTargetAsyncOption();
+                Results res = await this.MICommandFactory.ListTargetFeatures();
+                if (res.Contains("async"))
```

Comment:
`res?.Contains("async")`

Reply:
@amocz Please use the `?` operator to do null check:

`res?.Contains("async")` will do a null check on res and if that succeeds, it will then do a Contains("async") call. Your fix is missing the `?` operator.